rocketmq-connect-activemq/src/main/java/org/apache/rocketmq/connect/activemq/Config.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.rocketmq.connect.activemq;
+
+import java.lang.reflect.Method;
+import java.util.HashSet;
+import java.util.Set;
+
+import javax.jms.Session;
+
+import io.openmessaging.KeyValue;
+
+public class Config {
+
+	@SuppressWarnings("serial")
+	public static final Set<String> REQUEST_CONFIG = new HashSet<String>() {
+		{
+			add("activemqUrl");
+			add("destinationType");
+			add("destinationName");
+		}
+	};
+
+	public String activemqUrl;
+
+	public String activemqUsername;
+
+	public String activemqPassword;
+
+	public String destinationType;
+
+	public String destinationName;
+	
+	public String messageSelector;
+	
+	private Integer sessionAcknowledgeMode = Session.AUTO_ACKNOWLEDGE;
+
+	private Boolean sessionTransacted = Boolean.FALSE;
+	
+	public void load(KeyValue props) {
+
+		properties2Object(props, this);
+	}
+
+	private void properties2Object(final KeyValue p, final Object object) {
```

Comment:
properties2Object method is a tools method.You can refer to rocketmq,and put this class into a tools class!

Reply:
The same is true in Odar